### DISCUSSION TITLE
Create a develop and stable branch

### DISCUSSION THREAD

**Discussion by jl94x4** [lostb1t/replex#61] [Ideas]
I love Replex, and I will always contribute where I can to help improve it. But I've noticed recently, a lot of updates (active dev is brill) and some have broken quite a lot of functionality at times.

I would love it if you would have a stable and develop branch so when problems arise on the dev branch, we can switch back to the stable build without issues. 

I write this as my Plex is unusable due to the last update :(

**Comment by lostb1t** [comment]
A seperate develop won't help much untill i have some proper tests in place.

But I have added support for versioned images. So instead of latest you can use an versioned image and only upgrade when you want. Or rollback when necessary 

`ghcr.io/sarendsen/replex:0.14.6`
